aws_credentials.go

```diff
@@ -126,20 +126,32 @@ func process() {
 			kubeClient.Secrets(namespace.GetName()).Update(newSecret)
 		}
 
-		// Check if ServiceAccount has the imagePullSecrets
+		// Check if ServiceAccount exists
 		serviceAccount, err := kubeClient.ServiceAccounts(namespace.GetName()).Get("default")
 
 		if err != nil {
-			log.Fatalf("Could get find default service account!")
+			log.Fatalf("Couldn't get default service account!")
 		}
 
-		serviceAccount.ImagePullSecrets = []api.LocalObjectReference{{Name: *argDefaultSecretName}}
```

Comment:
This would conflict with other ImagePullSecrets used in the default service account since we're overwriting the field instead of appending to what already exists.